### PR TITLE
ENG-278 fix widget linting and bundle build result typos

### PR DESCRIPTION
**Pull request by jeffgo10**

### CODE REVIEW

File: generators/entity-server/templates/ui/widgets/formWidget/src/components/EntityForm.js.ejs [hunk 1]
```diff
@@ -47,8 +47,10 @@ const styles = theme => ({
     width: '100%',
   },
 });
-
-<%_ const ComponentName = `${entityClass}Form`; _%>
+<%_ 
+  const isFieldDateOrBlob = fieldsContainDate || fieldsContainBlob;
```
Comment (gidesan) [inline generators/entity-server/templates/ui/widgets/formWidget/src/components/EntityForm.js.ejs]: please use a similar naming, like `fieldsContainDateOrBlob`

File: generators/server/templates/prepareBundle.sh [hunk 2]
```diff
@@ -29,8 +29,10 @@ function injectResource() {
     local resource="$1"
     local destFile="$2"
 
+    local _NL=$'\\\n'
+
     echo "- Injecting resource $resource in $destFile"
-    sed -i 's|'"$INJECTION_POINT"'|'"$resource"'\n'"$INJECTION_POINT"'|g' "$destFile"
+    sed -i '.bak' 's|'"$INJECTION_POINT"'|'"$resource$_NL$INJECTION_POINT"'|g' "$destFile"
```
Comment (gidesan) [inline generators/server/templates/prepareBundle.sh]: what does this '.bak' change?

Comment (joewhite101) [reply generators/server/templates/prepareBundle.sh]: Required for the script to work on a mac.

File: generators/entity-server/templates/ui/widgets/formWidget/src/components/EntityForm.js.ejs (hunk 1 above)
Comment (jeffgo10) [reply generators/entity-server/templates/ui/widgets/formWidget/src/components/EntityForm.js.ejs]: done

**Pull request merged**